server/controllers/postExercise.js

```diff
@@ -0,0 +1,29 @@
+const Boom = require('@hapi/boom');
+const { addExercise } = require('../database/queries');
+const { exerciseValidation } = require('../utils');
+
+const postExercise = async (req, res, next) => {
+  const {
+    user: { id },
+  } = req;
+  const { exerciseDuration } = req.body;
+  const { exerciseID } = req.params;
+  try {
+    try {
```

Comment:
You might need to ensure that the userId is valid before going further (unless another idle ware has already done that prior to this controller running)